src/apps/frontend/contexts/auth.provider.tsx

```diff
@@ -43,6 +51,21 @@ const getAccessToken = (): AccessToken => JSON.parse(localStorage.getItem('acces
 
 const isUserAuthenticated = () => !!getAccessToken();
 
+const sendOTP = (countryCode: string, phoneNumber: string): Promise<AuthRecord> => {
+  // Backend Integration
+  console.log(countryCode, phoneNumber);
+  return null;
```

Comment:
This is incorrect.

You want to build the code all the way through the service layer and return mock response from there so that most of the code is ready.